src/main/java/com/spartronics4915/frc2019/Robot.java

```diff
@@ -20,8 +21,7 @@
 import java.util.jar.Attributes;
 import java.util.jar.Manifest;
 
-public class Robot extends IterativeRobot
-{
+public class Robot extends IterativeRobot {
```

Comment:
oops - our code formatting standard requires trailing braces

Reply:
many of the "noise" in this checkin appears to be due to the formatting changes.
I wonder if you can "fix" your checkin to eliminate that noise.  In other words,
we need to stick to one code formatting convention to make code reviews easier.